python/nav/etc/webfront/jwt.conf

```diff
@@ -3,6 +3,8 @@
 
 # The settings in the `nav` section are for tokens issued by NAV itself.
 # Comment this in and fill out the values if you want to activate local JWT tokens.
+# Token lifetime is set in seconds. The default is 1 hour for access tokens
+# and 1 day for refresh tokens.
```

Comment:
Sure.  You might also consider using https://github.com/Uninett/nav/blob/b7873039d5e33392119d3f366d685adc5d601811/python/nav/util.py#L416-L435
to allow for even more human-friendly input :)

Reply:
mmm very nice, I'll add this

Reply:
added this